keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -245,35 +247,35 @@ spec:
   provisioner: [4]
   releasePolicy: [5]
   accessMode: [6]
-  protocol: [7]
-  bucketClassName: [8]
-  bucketRef: [9]
+  bucketClassName: [7]
+  bucketRef: [8]
     name:
     namespace:
     uuid:
     resourceVersion:
-  secretRef: [10]
+  secretRef: [9]
     name:
     namespace:
-  azureBlobIdentity: [11]
+  azureBlobIdentity: [10] // TODO break into auth and conn
```

Comment:
Is this comment left over from me?

Reply:
no, just a reminded that i need to review the meeting recording - i remembered there was some talk about doing this.